### PR TITLE
Add description to plugins routes

### PR DESCRIPTION
**Pull request by soupette**
My PR is a:
💅 Enhancement

Main update on the:
Plugin

Add descriptions to the plugins' routes in order to make them appear in the upcoming documentation plugin.

### CODE REVIEW

File: packages/strapi-plugin-users-permissions/documentation/default/users-permissions-Role.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
```
Comment (lauriejim) [inline packages/strapi-plugin-users-permissions/documentation/default/users-permissions-Role.json]: @soupette I'm not sure this file has to be in this PR ?

File: packages/strapi-plugin-users-permissions/config/routes.json [hunk 2]
```diff
@@ -23,25 +27,32 @@
...
     },
     {
       "method": "GET",
-      "path": "/permissions",
+      "path": "/permissionss",
       "handler": "UsersPermissions.getPermissions",
       "config": {
         "policies": [],
-        "description": "Retrieve all the permissions of your project"
+        "description": "Fuck test",
```
Comment (maturanomx) [inline packages/strapi-plugin-users-permissions/config/routes.json]: 🚨 😱

**Comment by soupette** [comment]
@lauriejim this should be ready for your review

**Comment by lauriejim** [comment]
@soupette Maybe we should wait the Documentation plugin publication before merging this PR ?

**Comment by soupette** [comment]
I'd rather see it merged than being opened for months...

**Comment by lauriejim** [comment]
@soupette Can you please fix conflicts

**Comment by soupette** [comment]
@lauriejim you might be the best one to solve the conflicts on this file `packages/strapi-hook-mongoose/lib/index.js`

**Comment by lauriejim** [comment]
I close this PR, waiting for Documentation plugin.
Don't delete this branch.

**Pull request closed without merging**